### PR TITLE
add min() for Int and Float modules

### PR DESCRIPTION
**Pull request by bsnyder788**

### CODE REVIEW

File: gleam_stdlib/src/gleam/int.gleam [hunk 1]
```diff
@@ -16,3 +16,10 @@ pub fn compare(a, b) {
     }
   }
 }
+
+pub fn min(a, b) {
+  case a < b {
+  | True -> a
+  | False -> b
+  }
+}
```
Comment (lpil) [inline gleam_stdlib/src/gleam/int.gleam]: Could we have a newline at the end of this file please?

File: gleam_stdlib/test/gleam/float_test.gleam [hunk 2]
```diff
@@ -137,3 +137,26 @@ pub fn truncate_test() {
   |> float.truncate
   |> expect.equal(_, -7)
 }
+
+pub fn min_test() {
+    float.min(0., 0.)
```
Comment (lpil) [inline gleam_stdlib/test/gleam/float_test.gleam]: Can you indent with 2 spaces here and in the int test file please? Thank you

File: gleam_stdlib/src/gleam/int.gleam (hunk 1 above)
Comment (bsnyder788) [reply gleam_stdlib/src/gleam/int.gleam]: done

File: gleam_stdlib/test/gleam/float_test.gleam (hunk 2 above)
Comment (bsnyder788) [reply gleam_stdlib/test/gleam/float_test.gleam]: done

**Pull request merged**